api/v1alpha1/nodenetworkconfigurationenactment_types.go

```diff
@@ -20,7 +20,7 @@ type NodeNetworkConfigurationEnactmentList struct {
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:path=nodenetworkconfigurationenactments,shortName=nnce,scope=Cluster
-// +kubebuilder:printcolumn:name="Status",type="string",JSONPath=".status.conditions[?(@.type==\"Available\")].reason",description="Status"
+// +kubebuilder:printcolumn:name="Status",type="string",JSONPath=".status.conditions[?(@.status==\"True\")].type",description="Status"
```

Comment:
Isn't there a case when two conditions are True? I recall we originally considered allowing Progressing and Degraded being True at the same time, signalizing that the configuration (without rollout) is progressing, but some nodes have already failed. Is this still a possibility and what happens when we have 2x True?

Reply:
Only one is true at the same time https://github.com/nmstate/kubernetes-nmstate/blob/master/pkg/enactmentstatus/conditions/conditions.go, if it's more than one is true it will show the first one that contain Status=true.

I can leave this as it was before, the issue was that "MaxUnavailableNodeLimitReached" does not look good to state that the NNCE is kind of queued/pending.

Reply:
I like it better this way. Just making sure we won't end up in inconsistent and confusing state. If there is never double true, this should be fine.